### PR TITLE
added photo credit homepage

### PR DESCRIPTION
**Pull request by abdifitah12**

### CODE REVIEW

File: civictechprojects/static/css/partials/_LandingController.scss [hunk 1]
```diff
@@ -263,3 +263,6 @@
     }
   }
 }
+
```
Comment (marlonkeating) [inline civictechprojects/static/css/partials/_LandingController.scss]: Extra lines

File: civictechprojects/static/css/partials/_SplashScreen.scss [hunk 2]
```diff
@@ -71,3 +72,10 @@
     max-width: 700px;
   }
 }
+
+.photo{
```
Comment (marlonkeating) [inline civictechprojects/static/css/partials/_SplashScreen.scss]:
```suggestion
.photo-credit{
```

File: common/components/controllers/LandingController.jsx [hunk 3]
```diff
@@ -114,6 +114,7 @@ class LandingController extends React.PureComponent<{||}> {
         <SplashScreen className="LandingController-midsplash" header={header} text={text} img={HeroImage.MidLanding}>
           <Button variant="primary" className="LandingController-midsplash-btn SplashScreen-create-project-btn" href={url.section(Section.AboutUs)}>Learn More</Button>
         </SplashScreen>
+
```
Comment (marlonkeating) [inline common/components/controllers/LandingController.jsx]: Extra line

**Comment by abdifitah12** [comment]
i did change the css

**Comment by marlonkeating** [comment]
Closing due to ongoing design changes on the home page.

**Pull request closed without merging**